### PR TITLE
Fix the issue with mem > 100%

### LINKED ISSUE #151: kdash Crash Report
```
name = 'kdash'
operating_system = 'unix:Manjaro'
crate_version = '0.2.6'
explanation = '''
Panic occurred in file '~/.cargo/registry/src/github.com-1ecc6299db9ec823/tui-0.16.0/src/widgets/gauge.rs' at line 204
'''
cause = 'Ratio should be between 0 and 1 inclusively.'
method = 'Panic'
backtrace = '''

   0: 0x557115c570f2 - kdash::main::hbb7d0b00cd09bc2d
   1: 0x557115b1a0b3 - std::sys_common::backtrace::__rust_begin_short_backtrace::h37718759770cb226
   2: 0x557115c62eb7 - main
   3: 0x7f02e4dc4b25 - __libc_start_main
   4: 0x557115a9325e - _start
   5:        0x0 - <unresolved>'''
```

### PR DESCRIPTION
**Pull request by omid**
Fixes #151 
Fixes some tiny code styles
Replace `fold` with `map.sum` (If you don't like it, I can revert it)

### CODE REVIEW

**Review by deepu105 — APPROVED** [review]

File: src/cmd/mod.rs [hunk 1]
```diff
@@ -87,7 +87,7 @@ impl<'a> CmdRunner<'a> {
         if out.is_empty() {
           None
         } else {
-          Some(format!("v{}", out.replace("'", "")))
+          Some(format!("v{}", out.replace('\'', "")))
```
[the remark this answers is not in the document]
Comment (omid) [reply src/cmd/mod.rs:90]: clippy asked for it (nightly)

[the remark this answers is not in the document]
Comment (omid) [reply src/cmd/mod.rs:90]: https://rust-lang.github.io/rust-clippy/master/index.html#single_char_pattern

[the remark this answers is not in the document]
Comment (deepu105) [reply src/cmd/mod.rs:90]: ah ok, I'm on stable so not yet for it :P

**Comment by deepu105** [comment]
Thanks for this, will merge once green

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,31 +1,35 @@
...
+## Unreleased - 2022-XX-YY
+
+- Fix crashes when memory and/or cpu usages are higher than 100%
+
+## [0.2.6] - 2022-01-19
 
 - Fix status color of pods not ready
 
-## [0.2.5] - 21-Dec-2021
+## [0.2.5] - 2021-12-21
```
Comment (deepu105) [inline CHANGELOG.md:11]: why did you change the date format?
